pyproject.toml

```diff
@@ -1,3 +1,35 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "qtrack"
+authors = [
+    {name = "qlawton", email = "qlawton@ucar.edu"},
+]
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = ["License :: OSI Approved :: MIT License"]
+dynamic = ["version", "description"]
+requires-python = "~=3.8"
+dependencies = [
+    "cartopy",
+    "dill",
+    "imageio",
+    "joblib",
+    "matplotlib",
+    "netCDF4",
+    "numpy",
+    "pandas",
+    "scikit-learn",
+    "scipy",
+    "tropycal",
+    "xarray",
```

Comment:
@qlawton any dependency version constraints you are aware of, we can put them here and in the env yml.

Reply:
As of current version, seems to work with current versions of listed packages. I will update if I notice something does not work.